compiler/rustc_mir_transform/src/jump_threading.rs

```diff
@@ -17,6 +17,9 @@
 //!   if assigning anything different from `value` to `place` turns the `SwitchInt`
 //!   into `Goto { target }`.
 //!
+//! In this file, we denote as `place ?= value` the existence of a replacement condition
```

Comment:
Actually, it clearly must mean the former, because the notation makes no mention of the target, and therefore the actual effect is not specified and thus the polarity is irrelevant.